common/views/components/Buttons/Buttons.Dropdown.tsx

```diff
@@ -47,6 +47,7 @@ const Dropdown = styled(Space).attrs<{ $isTight: boolean }>(props => ({
     transform 350ms ease;
   border-radius: ${props => props.theme.borderRadiusUnit}px;
   box-shadow: ${props => props.theme.basicBoxShadow};
+  max-height: 400px;
```

Comment:
I maybe have a slight preference for something like `80vh`?

Reply:
I was wondering!